packages/panels/src/components/page-header/PageHeading.tsx

```diff
@@ -1,34 +1,40 @@
 import * as React from "react";
 import { ReactNode } from "react";
 import { Heading, HeadingVariant, Row } from "@stenajs-webui/core";
+import { IconDefinition } from "@fortawesome/fontawesome-svg-core";
+import { CircledIcon } from "@stenajs-webui/elements";
 
 export type PageHeadingVariant = "compact" | "standard" | "relaxed";
 
 interface PageHeadingProps {
+  icon?: IconDefinition;
   contentLeft?: ReactNode;
   contentRight?: ReactNode;
   heading: string;
   headingLevel?: HeadingVariant;
   variant?: PageHeadingVariant;
 }
 
-const variantToSpacing: Record<PageHeadingVariant, number> = {
-  compact: 1,
-  standard: 1.5,
-  relaxed: 2,
-};
-
 export const PageHeading: React.FC<PageHeadingProps> = ({
+  icon,
   heading,
   headingLevel = "h1",
```

Comment:
Maybe pageHeading should be `h2` if you have a Navbar heading?

Reply:
Right, I fix!